Resources/Prototypes/Body/Parts/animal.yml

```
@@ -44,7 +44,7 @@
     - state: r_hand
   - type: BodyPart
     partType: Hand
-    symmetry: Left
+    #symmetry: Left
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)